package.json

```diff
@@ -1,12 +1,13 @@
 {
     "name": "getrector-org",
     "dependencies": {
-        "@fortawesome/fontawesome-free": "^5.9.0",
-        "@fortawesome/free-brands-svg-icons": "^5.9.0",
-        "@fortawesome/free-regular-svg-icons": "^5.9.0",
-        "@fortawesome/free-solid-svg-icons": "^5.9.0",
+        "@fortawesome/fontawesome-free": "^6.2",
+        "@fortawesome/free-brands-svg-icons": "^6.2",
+        "@fortawesome/free-regular-svg-icons": "^6.2",
+        "@fortawesome/free-solid-svg-icons": "^6.2",
```

Comment:
Current deploy script on getrector.org doesn't run 

```
node install && yarn dev
```

so probably require to update after deployed

Reply:
Could you add that line to the deploy script now? 

So we don't have to think about it in the future and have it covered :+1:

Reply:
I can see `yarn build` in the script, is that same with `node install && yarn dev` ?

Reply:
I'm not much expert on this one :) 

Locally I've run:

```bash
yarn upgrade # similar to "composer update"
yarn dev # to build the assets
```

Reply:
It seems `npm install` actually needed, to download the updated packages first, I updated the script so it install the dependencies and build the webpack

Reply:
me too, I currently apply: `npm install && yarn build`, yarn build seems for production so it minified result on generated webpack, let's see if that's working on next build